effekt/shared/src/main/scala/effekt/generator/llvm/Transformer.scala

```diff
@@ -385,9 +385,14 @@ object Transformer {
         emit(callLabel(returnAddress, values.map(transform)))
         RetVoid()
 
-      case machine.NewStack(variable, prompt, frame, rest) =>
-        emit(Comment(s"newStack ${variable.name}"))
-        emit(Call(variable.name, Ccc(), transform(variable.tpe), newStack, List(transform(prompt))));
+      case machine.Reset(prompt, frame, rest) =>
+        emit(Comment(s"pushNewStack ${prompt.name}"))
```

Comment:
Very minor thing, the comments here and in `Resume` and in `Shift` still say "pushNewStack" etc.